client/src/partials/cash/cashboxes/cashboxes.js

```diff
@@ -75,14 +78,8 @@ function CashboxController($window, $uibModal, Session, Projects, Boxes, Currenc
       .then(function (data) {
 
         // workaround until we build a type column into the database.
-        // converts is_aux and is_bank columns into radio button select
-        if (data.is_auxillary) {
-          data.type = 'auxillary';
-        } else if (data.is_bank) {
-          data.type = 'bank';
-        } else {
-          data.type = 'primary';
-        }
+        // converts is_auxiliary into radio buttons
+        data.type = (data.is_auxiliary) ? 'auxiliary' : 'primary';
```

Comment:
I would recommend investigating `ng-true-value` and `ng-false-value` for driving these radio button states. 

I know this module will have to be re-factored to use `ui-router` very soon so this can be included in that.